src/general/OptionsMenu.cs

```diff
@@ -202,6 +216,12 @@ public class OptionsMenu : Control
     [Signal]
     public delegate void OnOptionsClosed();
 
+    public enum OptionsMode
+    {
+        MainMenu,
+        MicrobeStage,
```

Comment:
I think this should be named "InGame"

Reply:
I had it like that because I wasn't sure if the options would need to change when opened from another stage at some point.

Reply:
I don't foresee there being a need for stage specific options. It will be later confusing when each stage needs to say to the options "yeah, I'm the microbe stage (when I'm really the space stage)"
Even if there are stage specific options, they should all be always visible.